cdc_admin/cdc_admin.thrift

```diff
@@ -34,6 +34,9 @@ struct AddObserverRequest {
   # if this db is already being observed, an ALREADY_EXIST error is thrown
   1: required string db_name,
   2: required string upstream_ip,
+  # provide the upstream replicator's zk and helix cluster through request instead of command-line args
+  3: optional string replicator_zk_cluster,
+  4: optional string replicator_helix_cluster,
```

Comment:
Should we make these required?

Reply:
I think they should be optional so that we still support having them through the cluster command-line args. I think we should be conservative about adding required fields.